docs/log.org

```diff
@@ -28,6 +28,92 @@
   A time log can be found at Google Sheets: https://bit.ly/2PRKRl8
 
   The time log has exact tasks worked on each entry, for each
+* Week 14 and 15
+We felt it appropriate to spend more time on the report and demo parts of the
+project rather than documenting this log. Most of the past two weeks have been
+writing the report. That said, we are doing some developing and training in the
+background to try and fine tune our results, particularly audio quality.
+** Report
+To efficiently work on the report from home, we have many meetings and work in
+sprints. This means we spend one day of the week finalizing our changes
+(mondays), and one day to plan our sprints. Planning our sprints involves
+everyone (including supervisor) reading the report, commenting, followed by the
+group creating issues while discussing. Issues are simple, quick fixes that can
+usually be solved in a few hours.
+
+We started this around week 13, but now a few weeks later, it has definitely
+sped up writing the report. We also feel the quality of everything has gone up
+significantly and it makes it very easy to just pick something to work on,
+without needing to meet the rest of the group.
+
+As of right now, we are almost done with the report. We would have been done
+sooner if training wasn't a bottle neck for the results section, but not much
+can be done about that.
+
+** Exhibition
+Elias wrote and recorded a video for the exhibition in week 14. This allows us
+to solely focus on report and demo, which is great.
```

Comment:
Some things are not entirely done. I need to add actual audio samples, the profile pictures in the beginning should be replaced to be unique per person (people don't need pictures of themselves if they don't want through)), I should also do some takes and edit them so that I don't stumble over my words in the final version.